quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection/main.bicep

```diff
@@ -0,0 +1,213 @@
+@description('Specifies whether to deploy Azure Databricks workspace with secure cluster connectivity (SCC) enabled or not (No Public IP)')
+param disablePublicIp bool = false
+
+@description('Location for all resources.')
+param location string = resourceGroup().location
+
+@description('The name of the network security group to create.')
+param nsgName string = 'databricks-nsg'
+
+@allowed([
+  'trial'
+  'standard'
+  'premium'
+])
+@description('The pricing tier of workspace.')
+param pricingTier string = 'premium'
+
+@description('Cidr range for the private subnet.')
+param privateSubnetCidr string = '10.179.0.0/18'
+
+@description('The name of the private subnet to create.')
+param privateSubnetName string = 'private-subnet'
+
+@description('Cidr range for the public subnet..')
+param publicSubnetCidr string = '10.179.64.0/18'
+
+@description('The name of the public subnet to create.')
+param publicSubnetName string = 'public-subnet'
+
+@description('Cidr range for the vnet.')
+param vnetCidr string = '10.179.0.0/16'
+
+@description('The name of the virtual network to create.')
+param vnetName string = 'databricks-vnet'
+
+@description('The name of the Azure Databricks workspace to create.')
+param workspaceName string
+
+var managedResourceGroupName = 'databricks-rg-${workspaceName}-${uniqueString(workspaceName, resourceGroup().id)}'
+
+resource nsg 'Microsoft.Network/networkSecurityGroups@2020-05-01' = {
+  location: location
+  name: nsgName
+  properties: {
+    securityRules: [
```

Comment:
Thanks Stephen for removing excess variables in ARM template and making it more cleaner. We also don't NSG rules, can you remove them as well in ARM template and Bicep file as well?

Reply:
@mvvsubbu  Could you be more specific on what I should remove?  Note that everything in here was in the original sample.  thx

Reply:
@StephenWeatherford I am asking help to remove securityRules in NSG resources as we don't need them.

Reply:
The entire securityRules property?  I don't have any knowledge of the workings of this sample, I've only converted it.

@bmoore-msft can you help?

Reply:
Let's separate the two issues - this PR is just about bicep conversion, not "clean-up".  If we also want to clean up the sample, let's do a separate PR from a domain expert...

Reply:
Ok sure.